### PR TITLE
CP-121: Django 1.9 Upgrade (backwards compatible changes)

### PR DESCRIPTION
**Pull request by linseyray**
See an overview of the changes [here] (https://github.com/vikingco/mvne-platform/pull/4776).

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -23,7 +23,7 @@
 
 setup(
     name='gargoyle',
-    version='0.11.0',
+    version='0.11.1',
```
Comment (pieterdd) [inline setup.py]: Not sure if this is the right way to handle semantic versioning for forked repos. @nielsvanoch, do you know?

File: gargoyle/manager.py [hunk 2]
```diff
@@ -152,7 +152,7 @@ def as_request(self, user=None, ip_address=None):
 if hasattr(settings, 'GARGOYLE_CACHE_NAME'):
     gargoyle = SwitchManager(Switch, key='key', value='value', instances=True,
                          auto_create=getattr(settings, 'GARGOYLE_AUTO_CREATE', True),
-                         cache=get_cache(settings.GARGOYLE_CACHE_NAME))
+                         cache=caches[settings.GARGOYLE_CACHE_NAME])
```
Comment (pieterdd) [inline gargoyle/manager.py]: Not sure if this is a correct substitution. Can you doublecheck?

Comment (linseyray) [inline gargoyle/manager.py]: I double-checked and tested both in shell_plus; caches is just another way to obtain the cache instance, they are directly interchangeable.

**Comment by WimCrols** [comment]
:shipit:

**Pull request merged**